### ISSUE TITLE
Sensor connections

### ISSUE THREAD

**Issue by skuklinsky** [engineering-11/ParticulateMatter#1]
Can you edit the readme to include how to connect the sensor?

Also, with the GPS sensor connected, running the script prints out a false value around 11,000. Since something is connected to USB it still reads the bits and prints them, but it doesn't know that there's no PM sensor attached

**Comment by luitelshreejal** [comment]
I agree with Stephen. It would be helpful to edit the Readme to show how the sensor should be connected. A **picture** would be best. 

We retrieved values, however, they were either 0, 256, 512, 768. There might be a problem with your code. This might be an issue with how it's translating the bits.

**Comment by skuklinsky** [comment]
There's not really a problem with the code. It's just that when a non-PM sensor is connected, the code reads data as if it's from a PM sensor

**Comment by skuklinsky** [comment]
@luitelshreejal can you describe what the wire connections are between the pins and the sensor adaptor?

**Comment by luitelshreejal** [comment]
Yeah, black = GND, RED = VCC, green = RX, white = TX.

**Comment by luitelshreejal** [comment]
@skuklinsky
